djmoney/money.py

```diff
@@ -24,8 +24,21 @@ class Money(DefaultMoney):
 
     def __init__(self, *args, **kwargs):
         self.decimal_places = kwargs.pop("decimal_places", DECIMAL_PLACES)
+        self._decimal_places_display = kwargs.pop("decimal_places_display", None)
```

Comment:
Probably having `decimal_places_display` in the signature will make this argument more explicit (also, IDEs might pick it up). E.g.

```suggestion
    def __init__(self, *args, decimal_places_display=None, **kwargs):
        self.decimal_places = kwargs.pop("decimal_places", DECIMAL_PLACES)
        self._decimal_places_display = decimal_places_display
```

I believe it should work on the supported Python versions

Reply:
good idea - changed it